<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageFFScenario</class>
 <widget class="QWidget" name="PageFFScenario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="9" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Unknown C</string>
     </property>
     <property name="buddy">
      <cstring>unkC</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="enableAmmoCrates">
     <property name="toolTip">
      <string>Control whether Spartans have access to ammo crates that will refill any UNSC non-power weapons.</string>
     </property>
     <property name="text">
      <string>Ammo Crates</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Generator Fail Condition</string>
     </property>
     <property name="buddy">
      <cstring>generatorFailCondition</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="generatorFailCondition">
     <item>
      <property name="text">
       <string>Lose All</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lose One</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="generatorCount">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="specialValueText">
      <string>Disabled</string>
     </property>
     <property name="suffix">
      <string> generators</string>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="waveLimit">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="specialValueText">
      <string>no limit</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Unknown A</string>
     </property>
     <property name="buddy">
      <cstring>unkA</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Unknown B</string>
     </property>
     <property name="buddy">
      <cstring>unkB</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Generator Spawn Locations</string>
     </property>
     <property name="buddy">
      <cstring>generatorSpawnLocations</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="enableHazards">
     <property name="toolTip">
      <string>Control whether map-specific environmental hazards, like AI-controlled Banshees and Wraiths, are present.</string>
     </property>
     <property name="text">
      <string>Hazards</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="enableWeaponDrops">
     <property name="text">
      <string>Weapon Drops</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Generator Count</string>
     </property>
     <property name="buddy">
      <cstring>generatorCount</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="generatorSpawnLocations">
     <item>
      <property name="text">
       <string>Ordered</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Random</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Wave Limit</string>
     </property>
     <property name="buddy">
      <cstring>waveLimit</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Unknown D</string>
     </property>
     <property name="buddy">
      <cstring>unkD</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="unkA">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="unkB">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QSpinBox" name="unkC">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
     <property name="value">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QSpinBox" name="unkD">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
